web.py

```diff
@@ -45,9 +45,8 @@ def process_email_alerts():
         for balance in data:
             if balance.symbol == 'NZD':
                 if balance.available < app.config["MIN_AVAILABLE_NZD_BALANCE"]:
-                    balance_format = dasset.asset_dec_to_str(balance.symbol, balance.available)
```

Comment:
you dont want to remove the balance_format because it ensures the string is formatted to the correct number of decimal places

Reply:
re-added the balance_format and changed the msg.